components/local-app/BUILD.yaml

```diff
@@ -13,3 +13,13 @@ packages:
       image:
         - ${imageRepoBase}/local-app:${version}
         - ${imageRepoBase}/local-app:commit-${__git_commit}
+
+scripts:
+  - name: install-cli
```

Comment:
Could we do this automatically in the `.gitpod.yml` file for the initial install? Would maybe help with testing.

Reply:
We could add this to the README as well

Reply:
Yep, will do it 👍

Reply:
@filiptronicek  Updated README.md feel free to update it again XD